docs/decex/staking.md

```diff
@@ -7,17 +7,49 @@ keywords:
+2. At the end of an epoch (currently 100 blocks), the nominator is awarded their shares in the pool. The stake shares are the percentage of the total stake that is allocated to each nominator. The stake shares are used to calculate the share of operator's rewards the nominator is entitled to based on the amount they have staked and for how long. The stake shares are calculated as follows:
+    1. Compute the operator’s pool end-of-epoch $\text{shares\_per\_ssc}$ as the total number of shares divided by the sum of all stake in the pool and rewards gained during the previous epoch 
+
+    $\text{shares\_per\_ssc} = \text{total\_shares} / (\text{pool\_total\_stake} + \text{rewards}*(1-\text{nomination\_tax}))$.
+
+    2. Assign the $\text{shares}$ to this nominator based on the $\text{shares\_per\_ssc}$ of the pool 
+    
+    $\text{shares} = \text{deposit\_amount} * \text{shares\_per\_ssc}$.
+
+    3. The $\text{deposit\_amount}$ is added to $\text{pool\_total\_stake}$ of the operator pool and domain’s total stake.
+    4. The $\text{shares}$ of this nominator are added to $\text{total\_shares}$ of the pool.
+
+When the nominator decides to withdraw their stake, they submit a withdraw extrinsic. The withdraw extrinsic is processed at the end of the epoch and the stake is removed from the operator's pool and the domain's total stake. The nominator is then entitled to the rewards based on the stake shares and the amount of time they have staked.
+
+Operators can also withdraw their stake at any time by submitting a withdraw extrinsic. If their withdrawal leaves their stake below the domain's minimum requirements, the operator will be removed from the domain and their stake and the stakes of all nominators will be returned to their wallets.
+
+### Example
+
+Operator $O$ has staked 100 SSC and registered as an operator with minimum nominator stake of 10 SSC and nomination tax of 5%. Operator $O$ has 2 nominators $N1$ and $N2$ each staked 50 SSC. Initially $\text{shares\_per\_ssc} = 1$, so $O$ gets 100 shares, and $N1$ and $N2$ each get 50 shares and $\text{total\_shares}=100+50+50=200$. 
+
+In the next epoch, the pool has earned 10 SSC of rewards, and the operator took 5% of the rewards as commission (0.5 SSC). The pool end-of-epoch $\text{shares\_per\_ssc}$ is now $200/(200 + 10 * (1-0.05)) = 0.954654$. If a new nominator $N3$ stakes 30 SSC, the $\text{shares}$ they will get is $(30 * 0.954654) = 28$.
+
+Suppose after some time $\text{shares\_per\_ssc}$ value of this pool becomes 0.9. If $N1$ wants to withdraw 20 SSC, the corresponding shares will be deducted from their stake and the pool's total stake based on $\text{shares\_per\_ssc} = 0.9$. After withdrawal, $N1$'s remaining shares are $50-20*0.9=32$. Because of the time $N1$ has staked, the price of their shares has increased, so they they only had to "sell" 18 shares to get 20 SSC back, instead of 20 shares.
```

Comment:
```suggestion
Suppose after some time the $\text{shares\_per\_ssc}$ value of this pool becomes 0.9. If $N1$ wants to withdraw 20 SSC, the corresponding shares will be deducted from their stake and the pool's total stake based on $\text{shares\_per\_ssc} = 0.9$. After withdrawal, $N1$'s remaining shares are $50-20*0.9=32$. Because of the time $N1$ has staked, the price of their shares has increased, so they they only had to "sell" 18 shares instead of 20 shares to get 20 SSC back.
```